### PR TITLE
[NEW] Add update date for RC Apps

### PR DESCRIPTION
**Pull request by itzzritik**
Part of https://github.com/WideChat/Rocket.Chat/issues/941

Screenshot:
image

### CODE REVIEW

File: client/views/admin/apps/AppDetailsPageContent.tsx [hunk 1]
```diff
@@ -66,6 +69,14 @@ const AppDetailsPageContent: FC<AppDetailsPageContentProps> = ({ data }) => {
 							</Box>
 						)}
 					</Box>
+					{updatedAt && (
+						<Box display='flex' flexDirection='row' color='hint' alignItems='center'>
+							<Box mie='x4'>
+								{t('Updated_at')} {': '}
+								{formatDateAndTime(updatedAt)}
```
Comment (Shailesh351) [inline client/views/admin/apps/AppDetailsPageContent.tsx]: To make this short we can do 

```
{`${t('Updated_at')}: ${formatDateAndTime(updatedAt)}`}
```

Comment (itzzritik) [reply client/views/admin/apps/AppDetailsPageContent.tsx]: Thanks, Done

**Review by Shailesh351 — APPROVED** [review]
Looks good to me

**Comment by ear-dev** [comment]
@Shailesh351 @ItzzRitik This PR needs the dependency for apps-cli, right?  Is it up to date?  If so I will merge.....  thanks!

**Comment by itzzritik** [comment]
@ear-dev No, it’s not dependent on Apps CLI

**Comment by ear-dev** [comment]
Sorry....  apps-engine?

**Comment by itzzritik** [comment]
@ear-dev That Pr is merged, so I think it is safe to merge this as well.

@Shailesh351 could you also confirm please.

**Comment by Shailesh351** [comment]
Yes @ear-dev @ItzzRitik I have merged that PR, it also requires updating build files. So I have updated latest builds files in my catch up PR.

This will be ready for testing after Server catch PR. I'm on it.

**Comment by ear-dev** [comment]
@Shailesh351 I think I need some clarification please......  I was hoping to merge this independently from the upstream catchup you are working on.  I believe it has a dependency on the apps-engine PR that @ItzzRitik made, and that you merged.  So this PR will need the latest dependency in package-lock.json.  I was trying to confirm that was the case.  

But, I also see that you have merged an upstream catchup in apps-engine as well.......  does that mean then that we cannot merge this PR until the upstream Rocket.Chat PR you are working on is in?

**Comment by Shailesh351** [comment]
> I was hoping to merge this independently from the upstream catchup you are working on. I believe it has a dependency on the apps-engine PR that @ItzzRitik made, and that you merged. So this PR will need the latest dependency in package-lock.json. I was trying to confirm that was the case.

@ear-dev Yes that's correct. That's the way we can test this PR.

> But, I also see that you have merged an upstream catchup in apps-engine as well....... does that mean then that we cannot merge this PR until the upstream Rocket.Chat PR you are working on is in?

Yes, I merged it because It was blocking me from creating PRs for compiler/cli/server. Ritik's PR was missing updated build files. So I thought of updating it in my catchup PR.

Sorry, It created this confusion. My bad.

I think you will still be able to test this one, by updating apps-engine dependency to `git+https://github.com/WideChat/Rocket.Chat.Apps-engine.git#feature/include_deploy_date_time_for_rcapps` branch (I have updated build files in this branch just now) Instead of `master` for now.

Again sorry it created this mess 😅 I'm also thinking of using specific version names(i.e 1.28.0) instead of the `master` branch. So that we can test the builds separately. We can release tags in our repos similar to upstream.

**Comment by Shailesh351** [comment]
@ItzzRitik I have merged your latest Apps-engine PR. Can you update the apps-engine dependency to the latest commit in this PR? Thanks!

**Pull request merged**